frontend/src/features/contacts/contact/create/validation.ts

```diff
@@ -98,3 +97,12 @@ export const validationSchema = Yup.object().shape({
     }),
   }),
 });
+
+export const PersonValidationSchema = baseSchema.shape({
```

Comment:
nit: PersonValidationSchema should be start with a capital letter since it does not define a class/interface

Reply:
See other validation schemas in the code, they start with uppercase letter. I'm fine renaming but was following pre-existing patterns

Reply:
yeah, still looks like bad styling to me, but if that's the pattern for the code then sure.